code/places-monitor-android/src/phone/java/com/adobe/marketing/mobile/PlacesActivity.java

```diff
@@ -62,6 +65,35 @@ public static boolean isFineLocationPermissionGranted() {
 		return permissionState == PackageManager.PERMISSION_GRANTED;
 	}
 
+	/**
+	 * Checks if the permission to access fine location in background is granted.
```

Comment:
`Checks if permission to access fine location while the app is in background has been granted.`